src/main/kotlin/io/github/inductiveautomation/kindling/MainPanel.kt

```diff
@@ -139,59 +148,91 @@ class MainPanel : JPanel(MigLayout("ins 6, fill")) {
         isVisible = Debug.currentValue
     }
 
-    private val menuBar = JMenuBar().apply {
-        add(
-            JMenu("File").apply {
-                add(openAction)
-                for (tool in Tool.tools) {
-                    add(
-                        Action(
-                            name = "Open ${tool.title}",
-                        ) {
-                            fileChooser.fileFilter = tool.filter
-                            fileChooser.chooseFiles(this@MainPanel)?.let { selectedFiles ->
-                                openFiles(selectedFiles, tool)
-                            }
-                        },
-                    )
+    private val fileMenu = JMenu("File").apply {
+        add(openAction)
+        for (tool in Tool.tools) {
+            add(
+                Action(
+                    name = "Open ${tool.title}",
+                ) {
+                    fileChooser.fileFilter = tool.filter
+                    fileChooser.chooseFiles(this@MainPanel)?.let { selectedFiles ->
+                        openFiles(selectedFiles, tool)
+                    }
+                },
+            )
+        }
+        if (!SystemInfo.isMacOS) {
+            addSeparator()
+            add(
+                Action("Preferences") {
+                    preferencesEditor.isVisible = true
+                    preferencesEditor.toFront()
                 }
-            },
-        )
+            )
+        }
```

Comment:
Most Windows applications seem to put preferences into File, just because it's usually the first menu.